packages/ref-core/tests/unit/test_metrics.py

```diff
@@ -70,23 +70,28 @@ def cmec_right_metric_dict():
                 "Ecosystem and Carbon Cycle": {"name": "Ecosystem and Carbon Cycle"},
                 "Hydrology Cycle": {"name": "Hydrology Cycle"},
             },
+            "statistic": {
+                "overall score": {"name": "overall sccore", "units": "-"},
+                "bias": {"name": "mean bias", "units": "inherit"},
+                "rmse": {"name": "root mean squre error", "units": "inherit"},
+            },
         },
         "RESULTS": {
             "E3SM": {
-                "Ecosystem and Carbon Cycle": {"overall score": 0.11, "bias": 0.56},
-                "Hydrology Cycle": {"overall score": 0.26, "bias": 0.70},
+                "Ecosystem and Carbon Cycle": {"overall score": 0.11, "bias": 0.56, "rmse": -999.0},
+                "Hydrology Cycle": {"overall score": 0.26, "bias": 0.70, "rmse": -999.0},
```

Comment:
Perhaps these changes can be reverted as we don't have the concept of a missing value anymore?